doc/org.eclipse.cdt.doc.user/topics_Getting_Started.xml

```diff
@@ -4,6 +4,7 @@
           <topic label="Preparing the Workbench" href="getting_started/cdt_w_prepare_workbench.htm"/>
           <topic label="Creating a simple application" href="getting_started/cdt_w_basic.htm"/>
           <topic label="Creating a Makefile project" href="getting_started/cdt_w_newproj.htm"/>
+          <topic label="Core Build System" href="html/core_build_system.html"/>
```

Comment:
Precise name/location of generated HTML will be different if merged after #995.

Reply:
I have rebased this on #995.